extra/exports

```diff
@@ -265,6 +265,7 @@ luaL_pushcdata
 luaL_pushint64
 luaL_pushuint64
 luaL_setcdatagc
+luaL_tocpointer
```

Comment:
AFAICS you use it to implement `luaL_isnull` in tests for https://github.com/tarantool/tarantool-ee/pull/830. I think it's better to make `luaL_isnull` and `luaL_pushnull` public and do it in a separate commit.

Reply:
Done.